docs/index.md

```diff
@@ -10,11 +10,21 @@ lastpage:
 
 introduction/install
 introduction/api
-environments/all-environments
 wrappers/wrappers
 examples/morl_baselines
 ```
 
+```{toctree}
+:hidden:
+:caption: Environments
+
+environments/discrete-state
+environments/continuous-state
+environments/image-state
+environments/mujoco
```

Comment:
Looks weird to have mujocos under continuous state AND mujoco. I see two ways of separating:
## By family
Mujocos, grid worlds (~all discrete), classical (mountain car, lunar lander), misc (minecart, Mario, highway)

## By obs spaces
Like you did here, but removing Mujocos

Reply:
do you have a preference?

Reply:
Family, as in Gym?

Reply:
It is better indeed